manageiq-operator/api/v1alpha1/helpers/miq-components/cr.go

```diff
@@ -116,7 +116,7 @@ func httpdImageNamespace(cr *miqv1alpha1.ManageIQ) string {
 
 func httpdImageTag(cr *miqv1alpha1.ManageIQ) string {
 	if cr.Spec.HttpdImageTag == "" {
-		return "latest"
+		return "2-el9"
```

Comment:
Do you think `2` is specific, or should I change to `2.4-el9`?

Reply:
Yeah this is fine